### DISCUSSION TITLE
App closes/crashes when trying to sign out of account

### DISCUSSION THREAD

**Discussion by AliNaqvi97** [community/community#50621] [Enterprise]
<details>
<summary>Trying to sign out of an account on iOS app doesn't work if the account has been suspended. One of my GitHub enterprise accounts is no longer active due to me not being in the enterprise any longer but the mobile app still has the account data but nothing works since account is suspended. When I try to remove the account by signing out (under "Manage Accounts" in settings) the app closes and/or crashes and the account still persists. I cannot remove the account even after uninstall as I assume due to iOS keychain retaining the data and logging me in by default upon uninstalling/reinstalling so there's no way for me to remove the old account.</summary>
GitHub for iOS, Version 1.105.0 (94356885)
Device: iPhone 12 Pro Max (iOS 16.3.1)
</details>

**Comment by aconsuegra** [comment]
Hi @AliNaqvi97, thanks for reporting this crash! I'll create an issue for our engineers, so hopefully we can ship a fix soon.
